### ISSUE TITLE
Add the GetPaths operation to the Python Shell

### ISSUE THREAD

**Issue by m607123** [gchq/gaffer-tools#241]
Adding the GetPaths operation to the Python shell.
For more information, see gchq/Gaffer#1301.

**Comment by m607123** [comment]
To be updated once gchq/Gaffer#1301 is merged in, requires altering as per the changes on Gaffer branch. (GetPaths is now GetWalks)

**Comment by p013570** [comment]
Will be merged into develop as part of #275.
